packages/api/src/routes/helpers/request-logger.ts

```diff
@@ -60,6 +62,22 @@ export const requestLogger = (req: Request, res: Response, next: NextFunction):
   });
 };
 
+function splitUrlToClientAndPath(url: string): { client?: string; path: string } {
```

Comment:
The reasoning behind this is because we add URLs for specific clients. /dash-oss/medical-api/v1 or /ehr/athena/medical-api/v1. So i added this function so that the path in posthog can be the same and we can just add the client as a property that we can filter against if we want

Reply:
this is a good function for some unit tests